core/state_connector.go

```diff
@@ -21,12 +22,8 @@ const (
 )
 
 var (
-	costonChainID   = new(big.Int).SetUint64(16) // https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-16.json
-	songbirdChainID = new(big.Int).SetUint64(19) // https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-19.json
-	flareChainID    = new(big.Int).SetUint64(14) // https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-14.json
-
 	costonActivationTime   = big.NewInt(time.Date(2022, time.February, 25, 17, 0, 0, 0, time.UTC).Unix())
-	songbirdActivationTime = big.NewInt(time.Date(2200, time.January, 1, 0, 0, 0, 0, time.UTC).Unix())
+	songbirdActivationTime = big.NewInt(time.Date(2022, time.March, 28, 15, 0, 0, 0, time.UTC).Unix())
```

Comment:
In Flare repo this is: `constants.SongbirdID: time.Date(2022, time.March, 28, 14, 0, 0, 0, time.UTC),`

I would have expected these to be the same or Coreth first before Flare or am I missing something?

Reply:
This is the state connector activation time, not the hard fork.